uber/templates/preregistration/confirm.html

```diff
@@ -29,7 +29,7 @@
 <input type="hidden" name="return_to" value="{{ return_to }}" />
 <input type="hidden" name="undoing_extra" value="{{ undoing_extra }}" />
 
```

Comment:
In other words, only show the warning if they're viewing this page rather than having failed to upload some data.